generator/src/main/kotlin/io/github/mscheong01/krotodc/util/FieldName.kt

```diff
@@ -50,6 +50,8 @@ internal fun fieldNameToJsonName(name: String): String {
             }
             result.append(ch)
             isNextUpperCase = false
+        } else if (i == 0) {
+            result.append(ch.lowercaseChar())
```

Comment:
this would indeed change protobuf Pascal case fields to be generated as kotlin camel case fields. LGTM

Reply:
This block of code is specifically designed to generate the 'correct' oneof field name associated with the oneof type. I am uncertain about its potential impact on other logic within the code.